### PR TITLE
Fixes the broken scripts in cluster/centos

### PR DESCRIPTION
**Pull request by xiaoping378**
Fix the deploying scripts in cluster/centos, now it is broken.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33100)

### CODE REVIEW

**Comment by qdsang** [comment]
👍

**Comment by xiaoping378** [comment]
Hi @mikedanese , Please check it.

**Comment by xiaoping378** [comment]
Hi @eparis
please merge this.

File: cluster/centos/config-default.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ export SERVICE_CLUSTER_IP_RANGE=${SERVICE_CLUSTER_IP_RANGE:-"192.168.3.0/24"}
 export FLANNEL_NET=${FLANNEL_NET:-"172.16.0.0/16"}
 
 # Admission Controllers to invoke prior to persisting objects in cluster
-# If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
+# If we included ResourceQuota, we should keep it at the end of the list to prevent incremeting quota usage prematurely.
```
Comment (eparis) [inline cluster/centos/config-default.sh]: why did you add a typo?

File: cluster/centos/node/scripts/docker.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ DOCKER_OPTS=${1:-""}
 DOCKER_CONFIG=/opt/kubernetes/cfg/docker
 
 cat <<EOF >$DOCKER_CONFIG
-DOCKER_OPTS="-H tcp://127.0.0.1:4243 -H unix:///var/run/docker.sock -s devicemapper --selinux-enabled=false ${DOCKER_OPTS}"
+DOCKER_OPTS="-H tcp://127.0.0.1:4243 -H unix:///var/run/docker.sock -s overlay --selinux-enabled=false ${DOCKER_OPTS}"
```
Comment (eparis) [inline cluster/centos/node/scripts/docker.sh]: Is a non-POSIX compatible filesystem the default elsewhere?

File: cluster/centos/master/scripts/flannel.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+# Store FLANNEL_NET to etcd.
+attempt=0
+while true; do
+  /opt/kubernetes/bin/etcdctl --no-sync -C ${ETCD_SERVERS} \
+    get /coreos.com/network/config >/dev/null 2>&1
+  if [[ "$?" == 0 ]]; then
+    break
+  else
+    if (( attempt > 600 )); then
```
Comment (eparis) [inline cluster/centos/master/scripts/flannel.sh]: is 30 minutes really required?  Shouldn't this normally work on the very first attempt?

File: cluster/saltbase/salt/generate-cert/make-ca-cert.sh [hunk 4]
```diff
@@ -83,7 +83,7 @@ cd "${tmpdir}"
 if [ -f ~/kube/easy-rsa.tar.gz ]; then
 	ln -s ~/kube/easy-rsa.tar.gz .
 else
-	curl -L -O https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz > /dev/null 2>&1
+	curl -k -L -O https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz > /dev/null 2>&1
```
Comment (eparis) [inline cluster/saltbase/salt/generate-cert/make-ca-cert.sh]: Why? No.

**Review by eparis — COMMENTED** [review]
meaningless nits except the change to `curl -k`

**Comment by eparis** [comment]
ok to test

File: cluster/centos/config-default.sh (hunk 1 above)
Comment (xiaoping378) [inline cluster/centos/config-default.sh]: sorry, i run the rebase error....

File: cluster/centos/node/scripts/docker.sh (hunk 2 above)
Comment (xiaoping378) [inline cluster/centos/node/scripts/docker.sh]: run on centos7.2, the devicemapper will run failed.

File: cluster/centos/master/scripts/flannel.sh (hunk 3 above)
Comment (xiaoping378) [inline cluster/centos/master/scripts/flannel.sh]: i just follow cluster/centos/node/scripts/flannel.sh

File: cluster/saltbase/salt/generate-cert/make-ca-cert.sh (hunk 4 above)
Comment (xiaoping378) [inline cluster/saltbase/salt/generate-cert/make-ca-cert.sh]: yes, i will discard it.
but in my env, without -k, it will be failed.

File: cluster/centos/config-default.sh [hunk 5]
```diff
@@ -42,11 +42,11 @@ export FLANNEL_NET=${FLANNEL_NET:-"172.16.0.0/16"}
 
 # Admission Controllers to invoke prior to persisting objects in cluster
 # If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-export ADMISSION_CONTROL=NamespaceLifecycle,NamespaceExists,LimitRanger,ServiceAccount,SecurityContextDeny,DefaultStorageClass,ResourceQuota
+export ADMISSION_CONTROL=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota
```
Comment (eparis) [inline cluster/centos/config-default.sh]: Why not the same defaults as everyone else?
```
NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota
```

(NamespaceExists definitely needs dropped, agreed)

**Comment by xiaoping378** [comment]
Hi @eparis , when i rebase, i may make a mistake...
if necessary, please check this PR #33578 .

File: cluster/centos/config-default.sh (hunk 5 above)
Comment (xiaoping378) [inline cluster/centos/config-default.sh]: follow this, 
http://kubernetes.io/docs/admin/admission-controllers/ #Is there a recommended set of plug-ins to use

File: cluster/centos/node/scripts/docker.sh (hunk 2 above)
Comment (xiaoping378) [inline cluster/centos/node/scripts/docker.sh]: however, why we can't change it to be the same as coreOS's docker storage driver ?
